packages/xchain-thorchain/src/types/messages.ts

```diff
@@ -1,18 +1,27 @@
-import { AccAddress, Msg } from 'cosmos-client'
-import { StdTxFee } from 'cosmos-client/api'
-import { StdSignature } from 'cosmos-client/x/auth'
+import { cosmosclient } from 'cosmos-client'
+import { Coin } from 'cosmos-client/cjs/openapi/api'
 
 export type MsgCoin = {
   asset: string
   amount: string
 }
+export class Msg {}
+
+export type StdSignature = {
+  pub_key: cosmosclient.PubKey
+  signature: string
+}
+export interface StdTxFee {
+  gas?: string
+  amount?: Array<Coin>
+}
 
 export class MsgNativeTx extends Msg {
```

Comment:
Similar question as before: Why is `MsgNativeTx` extending `Msg`, if `Msg` does nothing?